sidecar/kubernetes/cmd/main.go

```diff
@@ -104,13 +104,13 @@ func handler(req *restful.Request, resp *restful.Response) {
 	responseWithJson(resp, tenants)
 }
 
-//readiness
```

Comment:
Same problem.

Reply:
I think it's not a problem, it fixed by lint automatically and the between of doc commend and `//` should have a empty space